src/jolt/driver/joltcb.py

```diff
@@ -118,12 +125,41 @@ def __str__(self):
         return self.args[1]
 
 class JOLTComputerBoard():
-    
     def __init__(self, simulated=False):
         self._ser_access = threading.Lock()
+        # Let's see if it actually raises
         self._serial = self._find_device(simulated=simulated)
         self.counter = 0
-        
+
+        # Version handling
+        be_hw_version = self.get_be_hw_version().lower()
+        if JOLT_V1_HW_IDENTIFIER in be_hw_version:
+            logging.info("JOLT V1 detected")
+            self.version = 2  # DEBUG
+        elif JOLT_V2_HW_IDENTIFIER in be_hw_version:
+            logging.info("JOLT V2 detected")
+            self.version = 2
+        else:
+            self.version = -1
+            logging.warning("Could not detect JOLT version")
+
+        # If JOLT V1 or unknown (probably also a V1)
+        if self.version <= 1:
+            self.voltage_range = (52, 70)
+        else:
+            # Set lower voltages for the JOLT V2.
+            self.voltage_range = (30, 37)  # Min useful / Max
+
+        self.voltage_forced_thread = None
+        self.stop_voltage_forced_event = threading.Event()
+
+    def sanitize(func):
+        """Wrapper to sanitize strings obtained over serial connection"""
+        def wrapper(self, *args, **kwargs):
```

Comment:
Please add a note explaining why this is needed. Does the function return these two characters *sometimes*, or *always*?